### PR TITLE
Manta M4P v2.2 support

### PR DESCRIPTION
**Pull request by asychev**
Complimentary PR https://github.com/Rat-OS/RatOS-configuration/pull/150

### CODE REVIEW

File: site/docs/boards/btt/manta-m4p-11.mdx
Comment (miklschmidt) [inline site/docs/boards/btt/manta-m4p-11.mdx:1]: Should be manta-m4p-10

File: site/docs/boards/btt/manta-m4p-22.mdx
Comment (miklschmidt) [inline site/docs/boards/btt/manta-m4p-22.mdx:1]: Should be manta-m4p-21

File: site/docs/boards/btt/manta-m4p-22.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: Manta M4P v2.2
```
Comment (miklschmidt) [inline site/docs/boards/btt/manta-m4p-22.mdx:2]: v2.1

File: site/docs/boards/btt/manta-m4p-22.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+import WiringSVG from "./_media/manta-m4p-22-wiring.svg";
```
Comment (miklschmidt) [inline site/docs/boards/btt/manta-m4p-22.mdx:5]: -21

File: site/docs/boards/btt/manta-m4p-22.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+# BIGTREETECH Manta M4P v2.2
```
Comment (miklschmidt) [inline site/docs/boards/btt/manta-m4p-22.mdx:7]: v2.1

File: site/docs/boards/btt/manta-m4p-22.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Wiring
+
+<WiringSVG width="100%" height={null} />
+
+[Open Full Size Image](_media/manta-m4p-22-wiring.svg)
```
Comment (miklschmidt) [inline site/docs/boards/btt/manta-m4p-22.mdx:13]: -21

File: site/docs/boards/list.md [hunk 5]
```diff
@@ -20,7 +20,8 @@ RatOS currently supports the following control boards:
 -   [BIGTREETECH Octopus Pro 429](btt/octopus-pro-429.md)
 -   [BIGTREETECH Octopus Pro H723 v1.0](btt/octopus-pro-h723.md)
 -   [BIGTREETECH Octopus Pro H723 v1.1](btt/octopus-pro-h723.md)
--   [BIGTREETECH Manta M4P](btt/manta-m4p.mdx)
+-   [BIGTREETECH Manta M4P v1.0](btt/manta-m4p-10.mdx)
+-   [BIGTREETECH Manta M4P v2.2](btt/manta-m4p-22.mdx)
```
Comment (miklschmidt) [inline site/docs/boards/list.md:24]: v2.1 and -21.mdx

File: site/docs/boards/btt/_media/manta-m4p-22-wiring.svg
Comment (miklschmidt) [inline site/docs/boards/btt/_media/manta-m4p-22-wiring.svg:1]: should be manta-m4p-21-wiring.svg

File: site/docs/boards/btt/manta-m4p-11.mdx
Comment (asychev) [reply site/docs/boards/btt/manta-m4p-11.mdx:1]: @miklschmidt Fixed

**Pull request merged**